.github/workflows/pr.yml

```diff
@@ -0,0 +1,14 @@
+name: "Aerie PR Logic"
```

Comment:
I'm wondering if we should just call this `Aerie PR Action` to reflect that is what is being used here.

Reply:
Good call, change has been made